src/strava_py/cli.py

```diff
@@ -0,0 +1,28 @@
+import argparse
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter
+    )
+    parser.add_argument("path", help="Input path to folder with GPX and / or FIT files")
+    parser.add_argument(
+        "-o", "--output_file", default="plot.png", help="Output PNG file"
```

Comment:
The dashes indicate an optional argument with two forms: short `-o` and long `--output_file`. If not specified, the default is used.